Fix the bug #527807.

This patch by Alexey Kryukov <anagnost@yandex.ru>.
This patch will be removed on next upstream release.

Index: fontforge-0.0.20090408/fontforge/fontviewbase.c
===================================================================
--- fontforge-0.0.20090408.orig/fontforge/fontviewbase.c
+++ fontforge-0.0.20090408/fontforge/fontviewbase.c
@@ -124,7 +124,7 @@
 		else {
 		    UnlinkThisReference(fv,fv->sf->glyphs[gid],fv->active_layer);
 		    for ( bdf=fv->sf->cidmaster?fv->sf->cidmaster->bitmaps:fv->sf->bitmaps; bdf!=NULL; bdf = bdf->next )
-			BCUnlinkThisReference( fv,fv->active_bitmap->glyphs[gid] );
+			BCUnlinkThisReference( fv,bdf->glyphs[gid] );
 		}
 	    } else if ( refstate == -1 )
     continue;
